torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -820,16 +835,61 @@ static std::vector<ExprHandle> toExprHandles(const std::vector<T>& sizes) {
   return dims;
 }
 
+std::vector<ExprHandle>& TensorExprKernel::getStridesForValue(
+    const torch::jit::Value* v) {
+  auto it = inputToStrides_.find(v);
+  if (it != inputToStrides_.end()) {
+    return it->second;
+  }
+  std::vector<ExprHandle> strides;
+  auto tt = v->type()->cast<TensorType>();
+  auto rank = tt->symbolic_sizes().rank();
+  auto concrete_strides = tt->strides().concrete_sizes();
+  if (concrete_strides) {
+    for (auto cs : *concrete_strides) {
+      strides.push_back(LongImm::make(cs));
+    }
+  } else {
+    throw std::runtime_error("Only concrete strides are handled");
```

Comment:
Nit: why not an assertion or TE error?

Reply:
I didn't have a preference either way. Changed it to an assertion.